corehq/apps/analytics/static/analytics/js/logging.js

```diff
@@ -0,0 +1,176 @@
+/* globals _, JSON */
+/**
+ * To Use this library, enable the feature flag ANALYTICS_NEW.
+ *
+ * To enable logging, enable the following INTERNAL feature flags to see each level:
+ * DEBUG: ANALYTICS_DEBUG
+ * VERBOSE: ANALYTICS_VERBOSE
+ * WARNING, DEPRECATED: ANALYTICS_DEPRECATED
+ *
+ */
+hqDefine('analytics/js/logging', function () {
+    'use strict';
+    var _LEVELS = {
```

Comment:
I like the styles for different levels. Do the levels work as expected with the web consoles in chrome and firefox? That is, can people use the web console to turn on/off different levels of logging?

<img width="780" alt="screen shot 2017-11-07 at 11 05 59 am" src="https://user-images.githubusercontent.com/1486591/32503619-b3e1ad78-c3ab-11e7-84c3-f846dba93c9f.png">

Reply:
unfortunately you can't apply the console levels to groups. also it feels like a good idea to completely turn off any console output altogether

Reply:
Argh, that's too bad about levels & groups. What do you mean about turning off console output altogether? There are other places, like formplayer, that log to console (granted, maybe they shouldn't).

Reply:
oh, nm, just saw comment below, looks like you mean turning it off within analytics